src/pages/Landingpage.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <q-page>
+    <div class="q-gutter-md">
+      <div class="row justify-between">
+        <q-parallax
+          src="~assets/SP logo44(cover).jpg"
+          class="cover-photo"
+          :height="screenHeight"
+        >
+          <h1 class="text-white">
+            Sining Pananadem
+            
+          </h1>
+        </q-parallax>
+      </div>
+    </div>
+  </q-page>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      screenHeight: 0
```

Comment:
you may want to use css for the height.. state are intended for business models..